maintenance.py

```diff
@@ -81,25 +104,26 @@ def wait_for_maintenance(callback):
             last_maintenance_event = maintenance_event
             callback(maintenance_event)
 
-
 def maintenance_callback(event):
     if event:
-        # print('Undergoing host maintenance: {}'.format(event))
+        logger.warning('Undergoing host maintenance: %s', event)
         # realistically, any sort of maintenence event should drain aerospike
         asinfo1 = [ASINFO, "-v", "quiesce:"]
         asinfo1.extend(args.options.split())
         call(asinfo1)
+        logger.debug('quiesce finished')
 
     else:
-        # print('Finished host maintenance')
+        logger.warning('Finished host maintenance')
```

Comment:
info not warning

Reply:
see above about info vs. warning from system maintenance perspective

Reply:
"Finished host maintenance" should be INFO.